src/ApiCompat/Microsoft.DotNet.ApiCompat.Shared/SuppressionFileHelper.cs

```diff
@@ -32,5 +32,25 @@ internal static class SuppressionFileHelper
                 log.LogMessage(MessageImportance.High, CommonResources.WroteSuppressions, suppressionOutputFile);
             }
         }
+
+        // Log whether or not we found breaking changes. If we are writing to a suppression file, no need to log anything.
+        public static void LogAPICompatSuccessOrFailure(bool generateSuppressionFile, ApiCompatServiceProvider serviceProvider)
```

Comment:
nit: Please don't flow the entire ApiCompatServiceProvider into this method. It has access to much more than what we need here. Instead consider just flowing the compatibility logger interface in.